source/rust_verify_test/tests/assoc_type_impls.rs

```diff
@@ -542,7 +542,7 @@ test_verify_one_file! {
         }
         impl<A: U> T for A where A::X: R { spec fn f() -> int { 100 } }
         impl U for char { type X = u8; }
-        impl U for bool { type X = u16; }
+        impl U for bool { type X = u8; }
```

Comment:
I think this should be kept as `X = u16` as it was before, but now the error is different.  I suggest:

```
proof fn test() {
    assert(<bool as T>::f() == 200);
    assert(<char as T>::f() == 100);
    assert(<bool as T>::f() == 100); // FAILS
    assert(<char as T>::f() == 200);
    assert(false);
}
```

Reply:
I see. I thought this was intended to check that verus reports conflicting trait implementations. I was confused because the above code doesn't actually have conflicting implementations but reported such an error due to what I assumed was a precision issue in the conflict detection logic in 1.82.

I'm happy with your suggestion too, to check that it doesn't get resolved to the wrong implementation.